nengo_loihi/loihi_cx.py

```diff
@@ -198,11 +198,12 @@ def _configure_filter(self, tau_s, dt):
                 "Current (U) scaling is required. Perhaps a synapse time "
                 "constant is too large in your model.")
 
-    def configure_lif(self, tau_rc=0.02, tau_ref=0.001, vth=1, dt=0.001):
+    def configure_lif(self, tau_rc=0.02, tau_ref=0.001, vth=1, dt=0.001,
+                      min_voltage=0):
```

Comment:
Since we're touching this, can we add a docstring and document these parameters?